src/annotations/modifiers/AnnotationItemModifier.cpp

```diff
@@ -162,6 +168,24 @@ void AnnotationItemModifier::updateCursor(Qt::CursorShape cursor)
 	}
 }
 
+void AnnotationItemModifier::checkDoubleClick()
```

Comment:
Would it have been possible to use the [sceneDoubleClickEvent](https://doc.qt.io/qt-5/qgraphicsscene.html#mouseDoubleClickEvent) and pass this information to the itemModifiert? The event would only be triggered on a double click and you could avoid checking for time yourself?